text/0249-v2-experiments.md

```diff
@@ -243,37 +243,47 @@ See Appendix D for an evaluation of all the possibilities.
     library work with a library that uses the stable library (no way to prevent the
     transitive dependency package with a different name from being installed).
 - Distinguish by prerelease tag
-  - Can't be used in **Maven**: an app can't depend on stable versions only
-    (a range of `[1.60.0,2.0.0)` would also match `1.61.0-experimental`).
+  - Most logical
```

Comment:
What's this line...? Did you forget to finish a sentence perhaps? 🙂

Reply:
It's just the 'most logical' :)